vocode/streaming/streaming_conversation.py

```diff
@@ -1004,15 +1004,15 @@ async def send_initial_message(self, initial_message: BaseMessage):
         )
         # TODO: configure if initial message is interruptible
         initial_message_tracker = asyncio.Event()
-        agent_response_event = (
-            self.interruptible_event_factory.create_interruptible_agent_response_event(
-                AgentResponseMessage(message=initial_message),
-                is_interruptible=False,
-                agent_response_tracker=initial_message_tracker,
+        # If it is outbound, update the history with the initial message
+        if self.agent.get_agent_config().call_type == CallType.OUTBOUND:
+            self.agent.update_history(
+                "human", self.transcriptions_worker.buffer.to_message()
             )
-        )
-        self.agent_responses_worker.consume_nonblocking(agent_response_event)
+            self.agent.update_history("message.bot", initial_message.text)
 
+        elif self.agent.get_agent_config().call_type == CallType.INBOUND:
+            self.agent.update_history("message.bot", initial_message.text)
```

Comment:
Should pass the `initial_message_tracker` to the update history function

Reply:
makes sense yeah